<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_ModInst_Ope</class>
 <widget class="QDialog" name="f_ModInst_Ope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modification de l'opération</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ressources.qrc">
    <normaloff>:/Images/Items/iconop.png</normaloff>:/Images/Items/iconop.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Prendre la valeur de </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CB_Variable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>I</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>J</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>O</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Q</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>S</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>U</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>W</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>et : </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="GrpB_ChoixAction">
       <property name="title">
        <string>Action</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="Radio_Affecter">
            <property name="text">
             <string>lui affecter la valeur</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Radio_Ajouter">
            <property name="text">
             <string>lui ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Radio_Soustraire">
            <property name="text">
             <string>lui soustraire</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="Radio_Inc">
            <property name="text">
             <string>l'incrémenter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Radio_Dec">
            <property name="text">
             <string>la décrémenter</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="SB_Valeur">
            <property name="maximum">
             <number>65025</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Radio_Affecter</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Valeur</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Radio_Ajouter</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Valeur</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Radio_Dec</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Valeur</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Radio_Inc</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Valeur</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Radio_Soustraire</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Valeur</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>f_ModInst_Ope</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>f_ModInst_Ope</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
